virtinst/domain/cputune.py

```diff
@@ -80,9 +88,11 @@ class DomainCputune(XMLBuilder):
     Class for generating <cpu> XML
     """
     XML_NAME = "cputune"
-    _XML_PROP_ORDER = ["vcpus", "cachetune", "memorytune", "vcpusched"]
+    _XML_PROP_ORDER = ["vcpus", "emulatorpin", "cachetune",
+            "memorytune", "vcpusched"]
 
     vcpus = XMLChildProperty(_VCPUPin)
+    emulatorpin = XMLChildProperty(_EmulatorPin, is_single=True)
```

Comment:
This is completely valid, but for this case it's a bit verbose. `<emulatorpin>` only has one child attribute, and can only be specified once, so it's less code to do:

```
emulatorpin_cpuset = XMLProperty("./emulatorpin/@cpuset")
```

the cli name should stay as `--cputune emulatorpin.cpuset` though. If `<emulatorpin>` ever grows more suboptions, we can split out an _EmulatorPin class like this to organize the code later and it won't affect the command line interface.

Reply:
Good to know. I've seen both styles around the rest of the code and I've generally tried following the style what was already there so far, but if that's the preference for "simple" suboptions, I'll stick to that unless there's a lot of suboptions.

Reply:
Next up will be iothreadpin[0-9]*. I've already implemented in a branch of my fork as an XMLChildProperty. Considering, it also only has 2 suboptions (`iothread`, and `cpuset), should I simplify that too before opening a pull request?